openassessment/xblock/grade_mixin.py

```diff
@@ -301,7 +302,10 @@ def has_feedback(assessments):
         if staff_assessment:
             median_scores = staff_api.get_assessment_scores_by_criteria(submission_uuid)
         elif "peer-assessment" in assessment_steps:
-            median_scores = peer_api.get_assessment_median_scores(submission_uuid)
+            median_scores = peer_api.get_assessment_scores_with_grading_strategy(
```

Comment:
The name of this variable would no longer be quite correct, but I understand that if we update it it would later affect the code.

Reply:
Yes, I'll need to refactor so that more generic variables can be used (the median_scores repeat quite a few times).